pkg/clusterresource/vsphere.go

```diff
@@ -111,7 +111,14 @@ func (p *VSphereCloudBuilder) addClusterDeploymentPlatform(o *Builder, cd *hivev
 }
 
 func (p *VSphereCloudBuilder) addMachinePoolPlatform(o *Builder, mp *hivev1.MachinePool) {
-	mp.Spec.Platform.VSphere = &hivev1vsphere.MachinePool{}
+	mp.Spec.Platform.VSphere = &hivev1vsphere.MachinePool{
+		NumCPUs:           2,
+		NumCoresPerSocket: 1,
+		MemoryMiB:         8192,
```

Comment:
Seems a little low, but is this an installer default?

Reply:
I got those values from the default machinesets created by the installer.

This seems to be where the default value is stored, but I'm not 100% sure on that:
https://github.com/openshift/installer/blob/83213bd69d8a3e3937e4cdd7f7d1e6b734a535bf/upi/vsphere/variables.tf#L159-L162

Reply:
Those are the defaults that I see, too.

https://github.com/openshift/installer/blob/a6cd8414b02ba8cac400b8d6628e11900ca31687/pkg/asset/machines/worker.go#L130-L139